mqtt-protocol/src/multithreadMain/kotlin/it/unibo/pulvreakt/mqtt/MqttProtocol.kt

```diff
@@ -1,62 +1,60 @@
+@file:Suppress("detekt:ImportOrdering")
```

Comment:
Please, do not suppress detekt warning, they are configured for a reason :)
Reorder the import in a lexicographic order.